unique_toolkit/pyproject.toml

```diff
@@ -0,0 +1,71 @@
+[tool.poetry]
+name = "unique_toolkit"
+version = "0.0.1"
+description = ""
+authors = [
+    "Unique Data Science <datascience@unique.ch>",
+]
```

Comment:
Is this a good idea? Might get spammed a lot. I mean people can open issues when there are encountering any problems.

Reply:
I don't believe this email exists at the moment. 🤷